src/Maestrano/packages.config

```diff
@@ -2,5 +2,5 @@
 <packages>
   <package id="Newtonsoft.Json" version="8.0.3" targetFramework="net45" />
   <package id="NUnitTestAdapter" version="2.0.0" targetFramework="net45" />
-  <package id="RestSharp" version="104.4.0" />
+  <package id="RestSharp" version="105.2.1" />
```

Comment:
Do you really need to change the RestSharp version ?
If not, we would prefer to keep it as a separate change.